config/jobs/README.md

```diff
@@ -1,9 +1,146 @@
+This document attempts to be a step-by-step easy copy-pasta guide to the use
+of prow jobs for the Kubernetes project. It may fall out of date. For more 
+info, it was sourced from the following:
+
+- [ProwJob docs](/prow/jobs.md)
+- [Life of a Prow Job](/prow/life_of_a_prow_job.md)
+- [Pod Utilities](/prow/pod-utilities.md)
+- [How to Test a Prow Job](/prow/build_test_update.md#How-to-test-a-ProwJob)
+
+### Job Types
+
+Generically, there are three typs of prow jobs:
+
+- **Presubumits** run against code in PRs
+- **Postsubmits** run after merging code
+- **Periodics** run on a periodic basis
+
+### Job Images
+
+Where possible, we prefer that jobs use images that are pinned to a specific
+version, containing only what is needed.
+
+Some good examples include:
+- [pull-release-unit] uses `golang:1.12` to run `go test ./...`
+- [pull-release-notes-lint] uses `node:11` to run `npm ci && npm lint`
+- [pull-org-test-all] uses `launcher.gcr.io/google/bazel:0.26.0` to run `bazel test //...`
+
+There are, unfortunately, many more bad examples out there:
+- `kubekins-e2e` is intended for e2e testing of kubernetes clusters, and has
+  "evolved organically" to put it charitably. There are tags built per release
+  of kubernetes, lots of deps and it's all quite brittle. It's tech debt we need
+  to pay down, and we really discourage its use.
+
+## Job Examples
+
+A presubmit that will run against all PRs to org/repo's master branch. It will
+run `make verify` in a checkout of `org/repo` at the PR's HEAD. It will report
+back to the PR via a status context named `pull-repo-dostuff`
+
+```yaml
+presubmits:
+  org/repo:
+  - name: pull-repo-dostuff  # convention: pull-(repo nam)-(suite name)
+    branches:
+    - master
+    decorate: true
+    always_run: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-in-go:v20190125-cc5d6ecff3
+        args:
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - --scenario=execute
+        - --
+        - make
+        - verify
+```
+
+## Adding or Updating Jobs
+
+1. Find or create the prowjob config file in this directory
+    - In general jobs for `github.com/org/repo` use `org/repo/filename.yaml`
+    - For kubernetes/kubernetes we prefer `kubernetes/sig-foo/filename.yaml`
+    - Ensure `filename.yaml` is unique across the config subdir; prow uses this as a key in its configmap
+1. Ensure an [`OWNERS`](https://go.k8s.io/owners) file exists in the directory for job, and has appropriate approvers/reviewers
+1. Write or edit the job config (please see [how-to-add-new-jobs](/prow/jobs.md#how-to-configure-new-jobs))
+1. Ensure the job is configured to to display its results in [testgrid.k8s.io]
+    - The new way: add [testgrid annotations]
+    - The old way: update [`testgrid/config.yaml`]
+    - Please see the [testgrid README](/testgrid/README.md) for more details on configuation options
```

Comment:
nb:  #13104 adds a "configurations.md" file that would be a better target for this link